### DISCUSSION TITLE
It is possible to use the API in a commercial application?

### DISCUSSION THREAD

**Discussion by psycodad** [openpotato/openplzapi.website#4] [Q&A] [answered]
Who is behind https://openplzapi.org/? 
Is it a good idea, to use the rest api on https://openplzapi.org/? Uptime guaranteed? 
Or ist the idea behind the project to host the rest api yourself?
I am a bit confused..

**Comment by fstueber** [comment, marked as answer]
Hi, thank your for your interest in our projects.

The OpenPLZ API project is developed and maintained by [STÜBER SYSTEMS](https://www.stueber.de/en/). We are a company from Germany.

Whether it is a good idea or not to use the API depends on your needs. This is an open data project that is still in its early stages. Any feedback is welcome. At the moment we cannot guarantee uptime, on the other hand the usage is free, even in a commercial application. And in the past, we have always reacted quite quickly when it comes to downtime.
